io-uring-test/src/main.rs

```diff
@@ -73,6 +73,9 @@ fn test<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
     #[cfg(feature = "unstable")]
     tests::queue::test_batch(&mut ring, &test)?;
 
+    // register
+    tests::register::test_register_files_sparse(&mut ring, &test)?;
```

Comment:
A cfg is needed here to ensure that tests can be performed when unstable is not enabled.